cypress/integration/common/kiali_login.ts

```diff
@@ -0,0 +1,52 @@
+import { Given, When, And, Then } from "cypress-cucumber-preprocessor/steps";
+
+const USERNAME = 'jenkins';
+const PASSWD = Cypress.env('passwd')
+const KUBEADMIN_IDP = 'my_htpasswd_provider';
+const auth_strategy = JSON.parse(Cypress.env('auth_strategy'))
+
+Given('user opens base url', () => {
+    cy.visit('/'); // visits baseUrl which is set in plugins/index.ts or bdd.ts
+    cy.window().then((win: any) => {
+        if (win = true) {
+            cy.log(win.localStorage + 'skipping login, console is running with auth disabled');
+            return;
+        }
```

Comment:
Not really sure what this is doing but it doesn't seem like it is necessary here. If we want to check if auth is disabled, can we just do `if auth_stragegy === 'anonymous'`?

Reply:
yes, you are right, it was form example. changed condition, fixed